var/sabath/assets/sabath/models/c/cosmoflow/cosmoflow_env.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash --login
+
+set -exo pipefail
```

Comment:
I don't see an shell pipes used in the code. Why is it here?

Reply:
Right, that got copied in from the horovod build script. I am changing it now.

Reply:
I think this is the only extraneous command: others looked legit.